packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -95,6 +99,21 @@ export const HelpCenterMoreResources = () => {
 						</a>
 					</div>
 				</li>
+				<li className="inline-help__resource-item">
+					<div className="inline-help__resource-cell">
+						<a
+							href={ `https://wordpress.com/home/${ siteSlug }?myHomeCoursePaymentsModal=payments-features` }
```

Comment:
This URL needs to be updated with the new param now

Reply:
Done